lib/components/PoolStats.jsx

```diff
@@ -0,0 +1,409 @@
+)
+
+const ReserveStat = (props) => (
+  <Stat
+    title={'Reserve'}
+    tokenSymbol={props.poolChainValues.token.symbol}
+    tokenAmount={props.poolChainValues.reserve.balance}
+    tooltip='Governance controlled funds contributing interest to the prize without being eligible to win'
+  />
+)
+
+const ReserveRateStat = (props) => (
+  <Stat
+    title={'Reserve rate'}
+    percent={props.poolChainValues.reserve.rate}
+    tooltip='Percent of each prize deposited into reserve'
+  />
+)
+
+const CUSTOM_YIELD_SOURCE_NAMES = {
+  1: {
+    '0x829df2cb6748b9fd619efcd23cc5c351957ecac9': 'rari'
+  }
+}
+
+const CUSTOM_YIELD_SOURCE_IMAGES = {
+  rari: '/custom-yield-source-images/rari.png'
+}
+
+const YieldSourceStat = (props) => {
+  const prizePoolType = usePrizePoolType()
+  const { chainId } = useNetwork()
+  const { data: prizePoolContracts } = usePrizePoolContracts()
+  const yieldSourceAddress = prizePoolContracts.yieldSource.address
+
+  let sourceImage, sourceName
+  if (prizePoolType === PRIZE_POOL_TYPE.compound) {
+    sourceName = 'Compound Finance'
+    sourceImage = CompSvg
+  } else if (prizePoolType === PRIZE_POOL_TYPE.yield) {
+    sourceName = CUSTOM_YIELD_SOURCE_NAMES?.[chainId]?.[yieldSourceAddress] || 'Custom Yield Source'
+    sourceImage = '/ticket-bg--light-sm.png'
+    const providedCustomImage = CUSTOM_YIELD_SOURCE_IMAGES[sourceName]
+    if (providedCustomImage) {
+      sourceImage = providedCustomImage
+    }
+  } else {
+    sourceName = <EmptyItem />
+  }
+
+  return (
+    <Stat
+      title='Yield Source'
+      value={
+        yieldSourceAddress ? (
+          <BlockExplorerLink address={yieldSourceAddress}>
+            <LinkIcon />
+          </BlockExplorerLink>
+        ) : null
+      }
+      sourceName={sourceName}
+      sourceImage={sourceImage}
+    />
+  )
+}
+
+const TimeUntilPrizeState = (props) => {
+  const { days, hours, minutes, seconds } = useTimeLeftBeforePrize()
+  return (
+    <Stat
+      title='Time to next prize'
+      value={<TimeDisplay days={days} hours={hours} minutes={minutes} seconds={seconds} />}
+    />
+  )
+}
+
+const ExitFeeStats = (props) => {
+  const { poolChainValues } = props
+  const [creditMaturationInDays, creditLimitPercentage] = getCreditMaturationDaysAndLimitPercentage(
+    poolChainValues.config.ticketCreditRateMantissa,
+    poolChainValues.config.ticketCreditLimitMantissa
+  )
+  return (
+    <>
+      <CreditLimitPercentageStat creditLimitPercentage={creditLimitPercentage} />
+      <CreditMaturationDaysStat creditMaturationInDays={creditMaturationInDays} />
+    </>
+  )
+}
+
+const CreditLimitPercentageStat = (props) => (
+  <Stat title='Early exit fee' percent={props.creditLimitPercentage} />
+)
+
+const CreditMaturationDaysStat = (props) => (
+  <Stat
+    title='Exit fee decay time'
+    value={`${props.creditMaturationInDays} day${props.creditMaturationInDays === 1 ? '' : 's'}`}
+  />
+)
+
+// APR Stats
+
+// TODO: Need to fetch data & token faucet address
```

Comment:
So I'm guessing this doesn't render currently? Or maybe we just need to remove the TODO ...

Reply:
Ah that can be removed